massgov/mayflower
# DP-7874 correct semantics of footer navigation

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


No visual change.

### Common changes to both patternlab and react:

- Remove `section.ma__footer-links` as it doesn't have its header.
- Merge 3 `<nav>` to 1, and change its class to `ma__footer-links`.  
- Add `aria-label="Footer navigation"` to `nav.ma__footer-links`.
- Put all `ul.ma__footer-links__items` in `nav.ma__footer-links`.

### Patternlab only changes

- Remove `aria-labeledby` from `nav`.
- Remove `h2`.
- Add `aria-label` to each `ul.ma__footer-links__items` as a replacement of `h2`.

 ### Patternlab only changes

- Add `div.ma__footer-links__nav` as a container for `h2` and `ul.ma__footer-links__items` unit.
- Keep `h2` for each `ul.ma__footer-links__items`.
- Remove `id` from `h2`.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-7874)
- [Github issue]()

## Steps to Test


1. Go to /patternlab/patterns/02-molecules-footer-links/02-molecules-footer-links.html.
2. Inspect the above markup change for the footer links.
3. Go to page samples to check any visual difference.
4. Go to molcules/footer-links in Storybook.
5. Inspect the above markup change for the footer links.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
